chain/network/src/peer_manager/peer_store/mod.rs

```diff
@@ -1,19 +1,16 @@
 use crate::blacklist;
 use crate::network_protocol::PeerInfo;
-use crate::store;
 use crate::time;
 use crate::types::{KnownPeerState, KnownPeerStatus, ReasonForBan};
 use anyhow::bail;
 use im::hashmap::Entry;
 use im::{HashMap, HashSet};
 use near_primitives::network::PeerId;
-use near_store::db::Database;
 use parking_lot::Mutex;
 use rand::seq::IteratorRandom;
 use rand::thread_rng;
 use std::net::SocketAddr;
 use std::ops::Not;
-use std::sync::Arc;
 
 #[cfg(test)]
```

Comment:
let's add a high level comment on what 'PeerStore' role is right now.

As AFAIK it only holds the stuff in memory now, right? so we should also mention what happens on restart.